test/testvideocapture.c

```diff
@@ -0,0 +1,787 @@
+        int seg = 0;
+        int val = *c;
+        if (val >= 'A' && val <= 'Z') {
+            val = 'a' + val - 'A';
+        }
+        switch (val) {
+            case '0': seg = 167345; break;
+            case '1': seg = 67; break;
+            case '2': seg = 16253; break;
+            case '3': seg = 16732; break;
+            case '4': seg = 4267; break;
+            case '5': seg = 14273; break;
+            case '6': seg = 145372; break;
+            case '7': seg = 167; break;
+            case '8': seg = 1234567; break;
+            case '9': seg = 123467; break;
+            case '.': seg = 8; break;
+            case '-': seg = 2; break;
+            case '_': seg = 3; break;
+            case '+': seg = 29; break;
+            case '|': seg = 9; break;
+            case '[': seg = 1453; break;
+            case ']': seg = 1673; break;
+            case 'a': seg = 124567; break;
+            case 'b': seg = 1234567; break;
+            case 'c': seg = 1453; break;
+            case 'd': seg = 167345; break;
+            case 'e': seg = 12345; break;
+            case 'f': seg = 1425; break;
+            case 'g': seg = 123457; break;
+            case 'h': seg = 24567; break;
+            case 'i': seg = 9; break;
+            case 'j': seg = 6735; break;
+            case 'k': seg = 24567; break;
+            case 'l': seg = 453; break;
+            case 'm': seg = 145679; break;
+            case 'n': seg = 14567; break;
+            case 'o': seg = 134567; break;
+            case 'p': seg = 14256; break;
+            case 'q': seg = 134567; break;
+            case 'r': seg = 145; break;
+            case 's': seg = 14273; break;
+            case 't': seg = 19; break;
+            case 'u': seg = 45367; break;
+            case 'v': seg = 45367; break;
+            case 'w': seg = 345679; break;
+            case 'x': seg = 24567; break;
+            case 'y': seg = 23467; break;
+            case 'z': seg = 16253; break;
+        }
+        while (seg) {
+            SDL_FRect r;
+            SDL_FPoint p1, p2;
+            int s = seg - 10 * (seg/10);
+            p1 = segments[s - 1][0];
+            p2 = segments[s - 1][1];
+            r.x = p1.x + x0;
+            r.y = p1.y + y0;
+            r.w = p2.x - p1.x;
+            r.h = p2.y - p1.y;
+            SDL_RenderRect(renderer, &r);
+            seg /= 10;
+        }
+        x0 += 12;
+        c++;
+    }
+}
+
+#if defined(__linux__) && !defined(__ANDROID__)
+static void load_average(float *val)
+{
+    FILE *fp = 0;
+    char line[1024];
+    fp = fopen("/proc/loadavg", "rt");
+    if (fp) {
+        char *s = fgets(line, sizeof(line), fp);
+        if (s) {
+            SDL_sscanf(s, "%f", val);
+        }
+        fclose(fp);
+    }
+}
+#endif
+
+int main(int argc, char **argv)
+{
+    SDL_Window *window = NULL;
+    SDL_Renderer *renderer = NULL;
+    SDL_Texture *texture = NULL;
+    int texture_updated = 0;
+    SDL_Event evt;
+    int quit = 1;
```

Comment:
Its value should be inverted, or renamed to e.g. `running`.

Reply:
thanks fixed!